pyg_lib/csrc/ops/matmul.cpp

```diff
@@ -57,9 +132,10 @@ class SegmentMatmul : public torch::autograd::Function<SegmentMatmul> {
       input_grad = _segment_matmul(grad_out, ptr, other_t);
     }
     if (torch::autograd::any_variable_requires_grad({other})) {
-      // TODO (matthias) implement backward pass for `other`.
+      variable_list grad_out_list = break_w_ptr(grad_out, ptr);
```

Comment:
We can use `split` for this - similar to https://github.com/pyg-team/pyg-lib/blob/master/pyg_lib/csrc/ops/cuda/matmul_kernel.cu#L132-L136.